### ISSUE TITLE
make dust master "track"-able (add .gitignore, put things under source control, etc.)

### ISSUE THREAD

**Issue by jlmitch5** [monome/dust#113]
I just did a sync from Norns onto a usb flash drive.  I went to this repo and noticed some updates to a few of the supplied scripts, so I went to check out fetching and pulling in latest master to get those.  Currently, it seems like a lot of the supplied files are not under source control, so the pull would not be clean:

![screen shot 2018-06-10 at 5 32 45 pm](https://user-images.githubusercontent.com/1342624/41206594-63a0562a-6cd4-11e8-80cd-677fad5e181e.png)

My thinking is that making sure stuff that we want under source control to be tracked (and things that we don't to be ignored) would fix this.  It would allow users to manually pull and stay up to date with master (as opposed to waiting for releases).  I think that having a more definitive release-cycle makes a lot of since for norns' firmware (with QA, beta, etc.), but less so for the type of stuff that dust holds.

**Comment by catfact** [comment]
that output seems anomalous; most of those things are under git control.

i wonder if you cloned the repo before ownership was transferred to `monome` and didn't update the remotes, or something like that?

i have this:
```

**Comment by jlmitch5** [comment]
thanks @catfact.  after some more investigation, the remote was correct (monome's fork was listed) and the head is at https://github.com/monome/dust/commit/49d3b9abc4c7ba55e3d44f4b41494793ef44c3eb

I can hard reset to the latest head.   But I'm not sure if the "sync from" usb step within norns is gonna mess things up.  I'm curious if norns will update based on what's in the flash drive (good) or try to duplicate things (potentially leaving things in a wonky state).

**Comment by catfact** [comment]
fair point - i think i saw some discussion of improving that behavior. but seems like an issue for the sync scripts, less than for the git repo _per se_

**Comment by jlmitch5** [comment]
> but seems like an issue for the sync scripts, less than for the git repo per se

Ah interesting, is there some other script-syncing conversation I'm missing here or on lines?  I tried searching around for that for a second before, might have missed that.

Definitely true that this doesn't necessarily need to be done through git updating with this repo.
